jkube-kit/resource/helm/src/main/java/org/eclipse/jkube/kit/resource/helm/HelmUploader.java

```diff
@@ -13,10 +13,12 @@
  */
 package org.eclipse.jkube.kit.resource.helm;
 
+import org.eclipse.jkube.kit.common.KitLogger;
+
 import java.io.File;
 import java.io.IOException;
 
 public interface HelmUploader {
   HelmRepository.HelmRepoType getType();
-  void uploadSingle(File file, HelmRepository repository) throws IOException, BadUploadException ;
+  void uploadSingle(KitLogger kitLogger, File file, HelmRepository repository) throws IOException, BadUploadException ;
```

Comment:
Considering the logger is used to publish low-level messages (which might not be informative to end users), maybe we want to reconsider passing in the logger at all.